frontstage/views/sign_in/sign_in.py

```diff
@@ -42,13 +49,6 @@ def login():
             error_message = error_json.get('data', {}).get('detail')
             if 'Unauthorized user credentials' in error_message:
                 return render_template('sign-in/sign-in.html', form=form, data={"error": {"type": "failed"}})
-            elif 'User account locked' in error_message:
```

Comment:
This means users now have no idea wether they can't access their account because it's locked or if they need to verify their account. Quite a big change in the user journey which might need more thinking.

Reply:
This is a decision that has been made by IA. We should not be giving them the info of if an account exists in the system.

Reply:
@tej99 It should now be treated as an invalid login.

Reply:
@tej99 This is being checked with Dan